ql/src/java/org/apache/hadoop/hive/ql/parse/SemanticAnalyzer.java

```diff
@@ -15789,12 +15789,22 @@ private ValidTxnWriteIdList getQueryValidTxnWriteIdList() throws SemanticExcepti
     return null;
   }
 
+  private Set<Long> getTransactionedTables() throws SemanticException {
```

Comment:
Typo: should be `Transactional` instead of `Transactioned`, and also the method name is confusing because it returns table IDs, not tables. I think `getTransactionalTableIDs` would be a more clear name.